dilqr_rs/dilqr_exact.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import time
+
+from dilqr_rs.dilqr import DiLQR
+
+class DiLQR_Exact(DiLQR):
```

Comment:
It would be nice if we could follow python's style convention and name classes using CamelCase, i.e. `DiLqrExact`. Pycharm put wiggly lines under the class name if it doesn't conform to the style convention :D

Reply:
Agreed on `DiLqrExact`